### PR TITLE
fix failures and new case for 4.6

### PR DESCRIPTION
**Pull request by lihongyan1**
Fixed failures and created 3 new cases for 4.6
OCP-32857 - List all alerts and could filter alerts by state and severity
OCP-32858 - List all silences and could filter silences by state
OCP-32859 - List all alerting rules and could filter rules by severity and state
Depend on PR:
https://github.com/openshift/cucushift/pull/7973
v3 job:
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/118995/console

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @yapei Please help review

File: lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -284,4 +291,46 @@ hide_show_alert_graph_button:
...
+    op: click
+clear_name_filter_if_exists:
+  params:
+    filter_category: Name
+  action:
+    if_element:
+      selector:
+        <<: *specific_filter
+    ref: clear_specifc_filter
+list_alerts_by_filters_clear:
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml:313]: the function name is not good

Comment (lihongyan1) [reply lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml:313]: any suggestion? here the filter work on alerts/silence/alert rule, all alerts related. I feel the name is ok

Comment (juzhao) [reply lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml:313]: ok

File: lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml [hunk 2]
```diff
@@ -24,10 +24,10 @@ check_div_text:
     selector:
       xpath: //div[contains(text(),'<text>')]
 filter_alert:
-  params:    
-    class_text: co-text-filter
+  params: 
+    class_text: co-text-filter
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml:28]: White spaces added at end of line

**Comment by akostadinov** [comment]
I fixed the white spaces. Thanks all.

/unhold
/lgtm

**Pull request merged**